clients/rust/Cargo.toml

```diff
@@ -23,13 +23,13 @@ rmp-serde = "1.0"
 serde = { version = "^1.0", features = ["derive"], optional = true }
 serde_json = "1.0"
 serde_with = { version = "^3.0", optional = true }
-solana-program = "> 1.14, < 1.18"
+solana-program = "> 1.14, < 1.19"
```

Comment:
Remind me why aren't just leaving it open to anything > 1.14 (no upper limit)?

Reply:
Mostly paranoia and no SemVer guarantee